source/Cake.ExtendedNuGet/Cake.ExtendedNuGet.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0;net461</TargetFrameworks>
```

Comment:
LGTM in principle, but I wonder if adding `net461` would give bring any benefit on this particular addin given that it depends on NuGet.Protocol and friends, and they target `net472` and `netstandard2.0` only. Not against it though.

Reply:
"only" following https://cakebuild.net/docs/extending/addins/best-practices#net-target-version here 👼